edbterraform/args.py

```diff
@@ -0,0 +1,260 @@
+import sys
+import os
+import argparse
+from pathlib import Path
+import textwrap
+from dataclasses import dataclass, field
+from collections import OrderedDict
+from typing import NoReturn
+
+from edbterraform.lib import generate_terraform
+from edbterraform.CLI import TerraformCLI
+from edbterraform import __project_name__
+from edbterraform.Logger import logger
+
+ENVIRONMENT_PREFIX = 'ET_' # Appended to allow overrides of defaults
+
+@dataclass
```

Comment:
CLI commands failing with `python3.6`.
`dataclasses` is builtin starting with `python3.7` so we need to update the dependencies.